.github/workflows/devtools.yml

```
@@ -0,0 +1,81 @@
+  pull_request: # run on all PRs, not just PRs to a particular branch
+
+jobs:
+  smoke-devtools:
+    runs-on: macos-latest
+
+    steps:
+    - name: Set $DEPOT_TOOLS_PATH
+      run: echo ::set-env name=DEPOT_TOOLS_PATH::$GITHUB_WORKSPACE/depot-tools
+    - name: Set $DEVTOOLS_PATH
+      run: echo ::set-env name=DEVTOOLS_PATH::$GITHUB_WORKSPACE/devtools-frontend
+    - name: Set $BLINK_TOOLS_PATH
+      run: echo ::set-env name=BLINK_TOOLS_PATH::$GITHUB_WORKSPACE/blink_tools
+    - name: Set $PATH
+      run: echo ::set-env name=PATH::$DEPOT_TOOLS_PATH:$PATH
+
+    - name: git clone
+      uses: actions/checkout@v2
+      with:
+        path: lighthouse
+    
+    - name: Cache depot tools, devtools, blink tools and content shell
+      uses: actions/cache@v2
+      with:
+        path: |
+          ${{ env.DEPOT_TOOLS_PATH }}
+          ${{ env.DEVTOOLS_PATH }}
+          ${{ env.BLINK_TOOLS_PATH }}
+          ${{ github.workspace }}/lighthouse/.tmp/chromium-web-tests/content-shells
+        key: ${{ runner.os }}-${{ hashFiles('lighthouse/lighthouse-core/scripts/chromium-web-tests/download-*') }}
+
+    - name: Use Node.js 10.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 10.x
+
+    - run: yarn --frozen-lockfile
+      working-directory: ${{ github.workspace }}/lighthouse
+    - run: yarn build-devtools
+      working-directory: ${{ github.workspace }}/lighthouse
+
+    - name: Download depot tools
+      run: bash $GITHUB_WORKSPACE/lighthouse/lighthouse-core/scripts/chromium-web-tests/download-depot-tools.sh
+    - name: Download DevTools Frontend
+      run: bash $GITHUB_WORKSPACE/lighthouse/lighthouse-core/scripts/chromium-web-tests/download-devtools.sh
+    - name: Download Blink Tools
+      run: bash $GITHUB_WORKSPACE/lighthouse/lighthouse-core/scripts/chromium-web-tests/download-blink-tools.sh
+    - name: Download Content Shell
+      run: bash $GITHUB_WORKSPACE/lighthouse/lighthouse-core/scripts/chromium-web-tests/download-content-shell.sh
+
+    - name: Run Web Tests
+      run: bash $GITHUB_WORKSPACE/lighthouse/lighthouse-core/scripts/chromium-web-tests/run-web-tests.sh
+
+    - name: Print diff
+      if: failure()
+      run: find "$GITHUB_WORKSPACE/lighthouse/.tmp/layout-test-results/retry_3" -name '*-diff.txt' -exec cat {} \;
+    - name: Upload results
+      uses: actions/upload-artifact@v2
+      if: failure()
+      with:
+        name: results
+        path: ${{ github.workspace }}/lighthouse/.tmp/layout-test-results
+
+    # TODO: https://github.com/EndBug/add-and-commit/issues/48
+    - name: Reset Results
+      if: failure() && github.actor != 'patrickhulce' && github.actor != 'brendankenny'
+      run: bash $GITHUB_WORKSPACE/lighthouse/lighthouse-core/scripts/chromium-web-tests/run-web-tests.sh --reset-results
+    - name: Commit new expectations
+      if: failure() && github.actor != 'patrickhulce' && github.actor != 'brendankenny'
+      uses: EndBug/add-and-commit@pr-support
```

The changes have been released with `v4.4.0`  (that can also be accessed via the `v4` and `latest` refs). Please update this so that I can delete the PR branch ;)

```suggestion
      uses: EndBug/add-and-commit@v4
```